libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -101,6 +107,8 @@ class AP_BattMonitor
         INA2XX                     = 21,
         LTC2946                    = 22,
         Torqeedo                   = 23,
+        PWM_VOLTAGE_ONLY           = 26,
+        PWM_VOLTAGE_AND_CURRENT    = 27,
```

Comment:
Can we just look at whether a pin has been supplied to work out which are relevant?